pages/[id]/bridge.tsx

```diff
@@ -231,9 +231,10 @@ const Bridge = () => {
                       className="mt-5 mx-auto flex flex-row items-center rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600 disabled:bg-gray-300 disabled:cursor-not-allowed"
                       onClick={handleBridge}
                       disabled={
-                        bridgeTarget === BridgeTarget.L2
+                        amount == '' ||
```

Comment:
My button isn't disabled on on load. Playing around with it locally `(amount === '0' || !amount)` worked for me

Reply:
Thanks! Added here: [d61ea5f](https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/113/commits/d61ea5f1d743c663636162492207a2de495d85c3)